lib/logger_json/formatters/google_cloud_logger.ex

```diff
@@ -54,6 +54,7 @@ defmodule LoggerJSON.Formatters.GoogleCloudLogger do
     |> FormatterUtils.maybe_put(:error, FormatterUtils.format_process_crash(md))
     |> FormatterUtils.maybe_put(:"logging.googleapis.com/sourceLocation", format_source_location(md))
     |> FormatterUtils.maybe_put(:"logging.googleapis.com/operation", format_operation(md))
+    |> JasonSafeFormatter.format()
```

Comment:
I think we can move this call up to the like 53, because we should not have "bad" metadata added by FormatterUtils. This would save a couple of iterations traversing those keys, what do you think?

Reply:
Yep, this makes sense, I was being overly cautious but agree it's not needed so have moved it up the chain.